runtime/conanfile.py

```diff
@@ -34,7 +34,7 @@ class CloeRuntime(ConanFile):
         "boost/[>=1.65.1]",
         "inja/[~=3.3.0]",
         "spdlog/[~=1.9.0]",
-        "incbin/[~=1.74.0]@cloe/stable",
+        "incbin/[~=0.88.0]@cloe/stable",
```

Comment:
What's the reason for this downgrade?

Reply:
I think I explained in the commit message.
It goes like this:

1. incbin doesn't version
2. We invent some versioning based on the number of commits
3. We originally did `1.{#commits}.{#our-changes}`
4. The starting `1` is a bad idea though, if incbin every starts versioning itself.
5. So we drop the major version to 0.

Reply:
So it's not a downgrade, but a re-versioning.